test/evergreen.yml

```diff
@@ -665,6 +689,34 @@ functions:
 #######################################
 
 variables:
+  # Configure flags for builtins.
+  - &configure_flags_with_builtins
+    HAVE_BUILTIN_EXTENSION_LZ4: -DHAVE_BUILTIN_EXTENSION_LZ4=1
```

Comment:
If you remove the previous changes and then you need to use posix_configure_flags to get these into cmake correctly.

Reply:
You can remove this update.